recipes/rapidsim/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "rapidsim" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gcowan/RapidSim/archive/v{{ version }}.tar.gz
+  sha256: a88f2a7c76fd5bbb0b16b9b3bcdfaedbee1e78a5e9be407aacc0600245f94468
+  patches:
+    - conda-packaging.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip_compile_pyc:
+    - "*/utils/compareParts.py"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - sed
+  host:
+    - root
+
+test:
+  commands:
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/B2Kee 1000 1
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/B2Kee 100000
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/Bs2Jpsiphi 100000
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/Bd2D0rho0 100000
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/Bs2D0Kpi 10000
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/D02Kpi 100000
+    - RapidSim.exe ${RAPIDSIM_ROOT}/validation/Lb2chicpK 100000
+
+    - mkdir plots
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"B2Kee\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bs2Jpsiphi\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bd2D0rho0\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bs2D0Kpi\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"D02Kpi\")"
+
+about:
+  home: https://github.com/gcowan/RapidSim
+  license: MIT
+  license_family: MIT
+  license_file: Licence.md
```

Comment:
I just want to make sure this is the correct file. Otherwise, LGTM!

Reply:
[It is](https://github.com/gcowan/RapidSim/blob/master/Licence.md)